core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/progress/listener/DefaultSdkRequestProgress.java

```diff
@@ -27,7 +27,7 @@
 /**
  * An SDK-internal implementation of {@link SdkRequestProgress}. This implementation acts as a thin wrapper around {@link
  * AtomicReference}, where calls to get the latest {@link #progressSnapshot()} simply return the latest reference, while {@link
- * SdkRequestProgressUpdater} is responsible for continuously updating the latest reference.
+ * ProgressUpdater} is responsible for continuously updating the latest reference.
```

Comment:
Typo ?

Reply:
SdkRequestProgressUpdater is incorrect. I had iniitally named it so, then changed the class name to ProgressUpdater (will be the next pr)